add_map_plugin.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+import subprocess
+
+def get_args():
+	# default arguments
+	if len(sys.argv) == 2:
+		default_args = {'bookstore':["./map_config/bookstore.rb", "aws-robomaker-bookstore-world"],\
+				'small_house':["./map_config/small_house.rb", "aws-robomaker-small-house-world"],\
+				'small_warehouse':["./map_config/small_warehouse.rb","aws-robomaker-small-warehouse-world"],\
+				'no_roof_small_warehouse':["./map_config/no_roof_small_warehouse.rb","aws-robomaker-small-warehouse-world"]
+				}
+		try:
+			world_name = sys.argv[1]
+			config_file = default_args[world_name][0]
+			world_file = "simulation_ws/src/deps/{0}/worlds/{1}.world".format(default_args[world_name][1], world_name)
+			output_file = "simulation_ws/src/cloudwatch_simulation/worlds/map_plugin.world"
+		except KeyError:
+			raise ValueError("Invalid argument")
```

Comment:
Add the message so the user can more easily debug. Look at `argparse` which does validation for you. `click` is a 3rd party CLI library, but easier to avoid installing additional packages for this simple use case